tests/StackExchange.Redis.Tests/SortedSets.cs

```diff
@@ -52,6 +61,153 @@ public class SortedSets : TestBase
+                Assert.Equal(5, diff);
+
+                var inter = db.SortedSetCombineAndStore(SetOperation.Intersect, destination, new RedisKey[]{ key1, key2});
+                Assert.Equal(5, inter);
+
+                var union = db.SortedSetCombineAndStore(SetOperation.Union, destination, new RedisKey[]{ key1, key2});
+                Assert.Equal(10, union);
+            }
+        }
+
+        [Fact]
+        public void SortedSetCombineErrors()
+        {
+            using (var conn = Create())
+            {
+                Skip.IfBelow(conn, RedisFeatures.v6_2_0);
+
+                var db = conn.GetDatabase();
+                var key1 = Me();
+                db.KeyDelete(key1, CommandFlags.FireAndForget);
+                var key2 = Me() + "2";
+                db.KeyDelete(key2, CommandFlags.FireAndForget);
+                var destination = Me() + "dest";
+                db.KeyDelete(destination, CommandFlags.FireAndForget);
+
+                db.SortedSetAdd(key1, entries);
+                db.SortedSetAdd(key2, entriesPow3);
+
+                // ZDIFF can't be used with weights
+                Assert.Throws<ArgumentException>(() => db.SortedSetCombine(SetOperation.Difference, new RedisKey[]{ key1, key2 }, new double[]{ 1, 2 }));
+                Assert.Throws<ArgumentException>(() => db.SortedSetCombineWithScores(SetOperation.Difference, new RedisKey[]{ key1, key2 }, new double[]{ 1, 2 }));
+                Assert.Throws<ArgumentException>(() => db.SortedSetCombineAndStore(SetOperation.Difference, destination, new RedisKey[]{ key1, key2 }, new double[]{ 1, 2 }));
+                // ZDIFF can't be used with aggregation
+                Assert.Throws<ArgumentException>(() => db.SortedSetCombine(SetOperation.Difference, new RedisKey[]{ key1, key2 }, aggregate: Aggregate.Max));
+                Assert.Throws<ArgumentException>(() => db.SortedSetCombineWithScores(SetOperation.Difference, new RedisKey[]{ key1, key2 }, aggregate: Aggregate.Max));
+                Assert.Throws<ArgumentException>(() => db.SortedSetCombineAndStore(SetOperation.Difference, destination, new RedisKey[]{ key1, key2 }, aggregate: Aggregate.Max));
+                // too many weights
+                Assert.Throws<StackExchange.Redis.RedisServerException>(() => db.SortedSetCombine(SetOperation.Union, new RedisKey[]{ key1, key2 }, new double[]{ 1, 2, 3 }));
+                Assert.Throws<StackExchange.Redis.RedisServerException>(() => db.SortedSetCombineWithScores(SetOperation.Union, new RedisKey[]{ key1, key2 }, new double[]{ 1, 2, 3 }));
+                Assert.Throws<StackExchange.Redis.RedisServerException>(() => db.SortedSetCombineAndStore(SetOperation.Union, destination, new RedisKey[]{ key1, key2 }, new double[]{ 1, 2, 3 }));
+            }
+        }
+
+        [Fact]
+        public void SortedSetIntersectionLength()
+        {
+            using (var conn = Create())
+            {
+                Skip.IfBelow(conn, RedisFeatures.v6_9_240);
```

Comment:
If you merge main in, this was changed to a 7 RC version so we don't have to visit these after adding :)